src/PKSim.Core/Snapshots/Mappers/CompoundPropertiesMapper.cs

```diff
@@ -199,6 +199,19 @@ private void updateAlternativeSelection(CompoundGroupSelection snapshotCompoundG
             return;
          }
 
+         var alternativeGroup = compoundProperties.Compound.ParameterAlternativeGroup(snapshotCompoundGroupSelection.GroupName);
```

Comment:
Adding some checks to get accurate warnings. Should they be Error? Not sure. Error Warning has no influence on what's happenign with the rest of the execution. THis is purely display